app/services/without_timestamps.rb

```diff
@@ -0,0 +1,11 @@
+module WithoutTimestamps
+  def self.without_timestamps
```

Comment:
`WithoutTimestamps.without_timestamps` seems a little unDRY - should we perhaps call this method something like `run`?

Actually what we have here is a service and perhaps we should try to have it confirm to service naming conventions so check out:

http://blog.silvabox.com/musings-on-service-objects/

so maybe:

```
ExecuteWithoutTimestamps.with(&block)
```

or maybe

```
WithoutTimetamps.run()
```

is better ...?

there's also a question of whether we should be instantiating an instance internally as we do with CreateProposedOrganisationEdit, but in this case that would seem like overkill, so I think just tweaking the method name might suffice ...